app/konnect/runtime-manager/consumer-groups.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Create Consumer Groups in Konnect
+content_type: tutorial
```

Comment:
I think this could be a how-to and it may be easier to strip some of the information. Otherwise, if you want to keep this to a tutorial you should write a concluding paragraph explaining what was done and how it can be changed.